addon/components/ember-table.js

```diff
@@ -19,7 +19,7 @@ const SELECTION_MODE_MULTIPLE = 'multiple';
 
 export default class EmberTable2 extends Component {
   @property attributeBindings = ['style:style'];
-  @property classNames = ['et-table'];
+  @property classNameBindings = ['tableClass'];
```

Comment:
Revert this, we should just allow users to style it themselves, no need to add custom class names